src/main/java/Anke.java

```diff
@@ -0,0 +1,66 @@
+import java.util.Scanner;
+
+public class Anke {
+    public static void main(String[] args) {
+        System.out.println("Hello! I'm Anke.");
+        System.out.println("What can I do for you?\n");
+        String line = "";
+        Task[] tasks = new Task[100];
+        int count = 0;
+        while (!line.equals("bye")) {
+            Scanner in = new Scanner(System.in);
+            line = in.nextLine();
+            Task task = null;
+            if (line.equals("bye")) {
+                break;
+            } else if (line.equals("list")) {
+                for (int i = 0; i < count; i++) {
+                    System.out.println((i + 1) + ". " + tasks[i].toString());
+                }
+                System.out.println("");
+                continue;
+            } else if (line.length() > 5 && line.substring(0, 5).equals("mark ")) {
+                int index;
+                try {
+                    index = Integer.parseInt(line.substring(5));
+                    if (index <= count && index > 0) {
+                        tasks[index - 1].setDone(true);
+                        System.out.println("Nice! I've marked this task as done:");
+                        System.out.println(tasks[index - 1].toString() + "\n");
+                        continue;
+                    }
+                }
+                catch (NumberFormatException e) {
```

Comment:
Could you change the formatting of this statement, such that the closing brace and the catch() condition are on the same line?